kuma/static/js/components/user-signup/signup.js

```diff
@@ -4,74 +4,73 @@
     var otherEmailInput = document.getElementById('id_other_email');
     var signupForm = document.getElementById('social-signup-form');
 
-    /**
-     * A simple test to determine if `onanimationend` is supported
-     * @returns {Boolean}
-     */
-    function isOnAnimationEndSupported() {
-        var elem = document.createElement('img');
-        return 'onanimationend' in elem;
-    }
-
     /**
      * Hides the specified static content container, shows the edit
-     * container and sets focus to the specified element.
+     * container and sets focus to input element.
      * @param {Object} staticContainer - The container to hide
-     * @param {Object} editContainer - The container to show
-     * @param {Object} focusElement - The element to focus
      */
-    function showEditFields(staticContainer, editContainer, focusElement) {
+    function showEditFields(staticContainer) {
         staticContainer.classList.add('hidden');
-        editContainer.classList.remove('hidden');
-        focusElement.focus();
+        staticContainer.nextElementSibling.classList.remove('hidden');
+        staticContainer.nextElementSibling.querySelector('input').focus();
     }
 
     /**
-     * Adds checkmark to MDN auth avatar, loads and animates the users Github avatar
-     * @param {HTMLFormElement} form - The form to submit after `onanimationend`
+     * Hide all containers with class `error` inside the `form`
+     * @param {Object} form - The form from which to clear visible error messages
      */
-    function animateAvatar(form) {
-        var avatarURL = event.target.dataset['avatar'];
-        var mdnProfileImgContainer = document.querySelector('.mdn-profile');
-        var mdnProfileImg = mdnProfileImgContainer.querySelector('img');
+    function clearErrors(form) {
+        var errors = form.querySelectorAll('.error');
 
-        mdnProfileImg.src = avatarURL;
-        mdnProfileImg.addEventListener('load', function() {
-            mdnProfileImgContainer.classList.add('checked');
-            mdnProfileImg.classList.add('animate');
-
-            mdnProfileImg.onanimationend = function() {
-                form.submit();
-            };
-        });
+        for (var i = 0, l = errors.length; i < l; i++) {
+            errors[i].classList.add('hidden');
+        }
```

Comment:
Man, I miss jQuery. But you should probably just do `errors.forEach(function(error) { error.classList.add('hidden') }` instead of this loop.

Reply:
Actually. Ignore that. It wouldn't really change anything.